File: pinyougou-manager-web/src/main/resources/static/js/service/specificationOptionService.js
//服务层
app.service('specificationOptionService',function($http){

    this.findAllSpecificationOption=function (page,size,searchEntity) {
        return $http.post("../specificationOption/findAllSpecificationOption?page="+page+"&size="+size,searchEntity)
    }

    this.findOne=function (id) {
        return $http.get("../specificationOption/findOne?id="+id)
    }

    this.del=function (ids) {
        return $http.post("../specificationOption/delSpecificationOption",ids)
    }

    this.add=function (searchEntity) {
        return $http.post("../specificationOption/addSpecificationOption",searchEntity)
    }

    this.update=function (searchEntity) {
        return $http.post("../specificationOption/updateSpecificationOption",searchEntity)
    }
});

File: interface/src/main/java/com/kgc/pinyougou/service/SpecificationOptionService.java
package com.kgc.pinyougou.service;

import com.kgc.pinyougou.domain.SpecificationOption;
import com.kgc.pinyougou.utils.PageResult;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface SpecificationOptionService {

	/**
	 * @Author Admin
	 * @description:  //TODO
	 * @param:
	 * @param pageNum   分页页数
	 * @param pageSize  分页记录数
	 * @return: com.kgc.pinyougou.utils.PageResult
	 * @time: 2020/4/25 22:11
	 */
	PageResult findAllSpecificationOption(Integer pageNum, Integer pageSize, SpecificationOption specificationOption);

	Integer insertSpecificationOption(List<SpecificationOption> specificationOptionList, Long specId);

	SpecificationOption findOne(Long id);

	Integer updateSpecificationOption(SpecificationOption specificationOption);

	Integer deleteSpecificationOption(Long [] ids);
}
